lib/loader.js

```diff
@@ -17,26 +16,26 @@ module.exports = function loader(content) {
 
   const done = this.async();
   const loaderContext = this;
-  const { resourcePath, resource, context, loaderIndex } = loaderContext;
+  const { resourcePath, context, loaderIndex } = loaderContext;
   // webpack 1 compat
   const resourceQuery = loaderContext.resourceQuery || '';
   const compiler = loaderContext._compiler;
   const isChildCompiler = compiler.isChild();
   const parentCompiler = isChildCompiler ? compiler.parentCompilation.compiler : null;
-  const issuer = loaderContext._module && loaderContext._module.issuer;
-  const matchedRules = getMatchedRules(resource, getLoadersRules(compiler), issuer);
 
   if (!content.includes('<svg')) {
-    throw new Exceptions.InvalidSvg(content, matchedRules);
+    throw new Exceptions.InvalidSvg(content);
   }
 
+  const configObj = { context: loaderContext };
+
+  configObj.config = getOptions(loaderContext);
+  configObj.target = context.target || loaderContext.target || (context.options && context.options.target);
```

Comment:
`(context.options && context.options.target)` is legacy and could be removed

Reply:
Removed.